nimbus.fml.yaml

```diff
@@ -299,34 +299,50 @@ features:
             "inactive-tabs": true,
           }
   contextual-hint-feature:
-    description: This set holds all features pertaining to the usage of contextual hints, whether it's to introduce new features or remind users of existing ones.
+    description: This set holds contextual hints that can be toggled to hide or show based on these settings.
     variables:
       features-enabled:
         description: This property provides a lookup table of whether specific contextual hints are enabled.
         type: Map<ContextualHint, Boolean>
         default:
         {
-          "toolbar-contextual-hint": true,
-          "jump-back-in-contextual-hint": true,
           "jump-back-in-synced-tab-contextual-hint": false
         }
     defaults:
       - channel: beta
         value: {
           "features-enabled": {
-            "toolbar-contextual-hint": true,
-            "jump-back-in-contextual-hint": true,
             "jump-back-in-synced-tab-contextual-hint": true
           }
         }
       - channel: developer
         value: {
           "features-enabled": {
-            "toolbar-contextual-hint": true,
-            "jump-back-in-contextual-hint": true,
             "jump-back-in-synced-tab-contextual-hint": true
           }
         }
+  contextual-hint-copy:
```

Comment:
This should not be a separate feature. This should be a variable under `contextual-hint-feature` as it's just configuration that's part of that feature.